tethysapp/temp_precip_trends/api_helpers.py

```diff
@@ -7,65 +7,46 @@
 import xarray as xr
 
 
-def get_data(variable, dataset, params):
+def get_data(variable, dataset, params, cum_sum=False, offset_dates=None, return_json=True):
```

Comment:
I'll turn this around and ask why we are passing things in through the param object. I've refactored so that the get_data function takes all the params explicitly. So only the controller is dealing with the param objects.